src/CryptoAxus.API/Controllers/ArtistController.cs

```diff
@@ -123,6 +121,32 @@ public async Task<IActionResult> PostArtist([FromBody] CreateArtistDto artist)
         return CreatedAtRoute("GetArtistById", new { id = response.Result?.Id }, response);
     }
 
+    /// <summary>
+    /// Returns artist by Wallet Address
+    /// </summary>
+    /// <param name="userWalletAddress" example="0x507f191e810c19729de860ea"></param>
+    /// <response code="200">Success response with 200 code and information message about update</response>
+    /// <response code="404">Not Found response with 404 code and information message</response>
+    /// <response code="400">Bad Request response with 400 code and information message</response>
+    /// <returns></returns>
+    [HttpGet("{userWalletAddress:alpha:required}/userWalletAddress", Name = "GetArtistByWalletAddressRequest", Order = 5)]
+    [RequiresParameter(Name = "userWalletAddress", Required = true, Source = OpenApiParameterLocation.Path, Type = typeof(string))]
+    [SwaggerRequestExample(typeof(GetArtistByWalletAddressRequest), typeof(GetArtistByWalletAddressRequestExample))]
+    [ProducesResponseType(typeof(GetArtistByWalletAddressResponse), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(NotFoundArtistByWalletAddressResponse), (int)HttpStatusCode.NotFound)]
+    [ProducesResponseType(typeof(BadRequestArtistByWalletAddressResponse), (int)HttpStatusCode.BadRequest)]
+    public async Task<IActionResult> GetArtistByWalletAddressRequest([FromRoute] string userWalletAddress)
+    {
+        var response = await Mediator.Send(new GetArtistByWalletAddressRequest(userWalletAddress));
+
```

Comment:
Why media type not provided as part of the query string and implementation for parsing the media type value not added. See `GetArtistById` method.

Reply:
Resolved